hw/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM ubuntu:jammy
 RUN apt-get update
 RUN apt-get install -y gcc make valgrind
 
-RUN useradd runner
+RUN useradd -u 1001 runner
```

Comment:
Hard-coded uid would cause users whose uid is not 1001 to get into trouble with the permission problem.